source/operating-a-service/incident-playbook.html.md

```diff
@@ -9,53 +9,62 @@ Self-organise to appoint:
-4. Invite the appropriate people from the contact lists to the incident channel.
+Complete the details in the IncidentBot template, and press Enter, which will automatically create a dedicated Slack channel for the incident.
 
-### 5. Provide a service update to users outside DfE (comms lead)
+### 4. Focus on fixing the incident (usually tech lead)
 
-The Teacher Services team maintains a publicly available [service status dashboard](https://teacher-services-status.education.gov.uk/). During an incident, the comms lead needs to explain what’s happening to users outside DfE. The comms lead will need a GitHub account to do this, or delegate updates to a colleague who has one.
+The tech lead should be focused on either fixing the incident themselves or giving direction to other developers to do so.
 
-The updates are managed via GitHub Actions and issues on the [teacher-services-upptime repository on GitHub](https://github.com/DFE-Digital/teacher-services-upptime). If a service’s automatic health check is failing continuously, an issue will be created within 5 minutes of the failure occurring and the dashboard will start reporting a service issue.
+If the incident is not of a technichal nature, for example, as the result of a data breach, the focus should still be on resolving the incident. 
 
-To update the dashboard:
+### 5. Determine who to contact and how (comms lead)
 
-1. Navigate to the appropriate incident issue on the [GitHub issues page](https://github.com/DFE-Digital/teacher-services-upptime/issues)
-2. Add a comment to the issue
+Determine who needs to be contacted, based on the incident priority and affected services. 
 
-### 6. Start the incident report (any incident lead)
+Use the contacts from the [Teacher services list](https://educationgovuk.sharepoint.com.mcas.ms/sites/teacher-services-infrastructure/Lists/Teacher%20services%20list/AllItems.aspx) and your service's contact list if you have one.
+
+It may be best to include critical user groups like lead providers, or other external organisations we work with. Make sure to include programme delivery managers, deputy directors and service owners in case of a P1 incident.
+
+You need to decide how to contact them as well. You might need to use email, Teams, Slack or a combination to make sure you reach everyone. You can also add them to the Slack incident channel. 
+
+### 6. Identify any other services involved (any leads)
+
+The leads should identify:
+
+- any upstream services (both inside and outside DfE) which could be contributing to the issue
+- any downstream services (both inside and outside DfE) likely to be affected by the issue and raise incidents where needed
+
+After doing this, you might realise you need to contact those teams as well. This is the responsibility of the comms lead.
```

Comment:
```suggestion
- any downstream services (both inside and outside DfE) likely to be affected by the issue

It may be necessary to contact the upstream/downstream service teams and raise incidents in those services. This is the responsibility of the comms lead.
```